src/tx-format/transaction.md

```diff
@@ -131,15 +131,25 @@ Creates a contract with contract ID as computed [here](../identifiers/contract-i
 ## TransactionMint
 
 The transaction is created by the block producer and is not signed. Since it is not usable outside of block creation or execution, all fields must be fully set upon creation without any zeroing.
-
-| name           | type                         | description                                          |
-|----------------|------------------------------|------------------------------------------------------|
-| `txPointer`    | [TXPointer](./tx-pointer.md) | The location of the `Mint` transaction in the block. |
-| `outputsCount` | `uint8`                      | Number of outputs.                                   |
-| `outputs`      | [Output](./output.md)`[]`    | List of outputs.                                     |
+This means that the transaction ID must also include the correct `txPointer` value, not zeroed out.
+
+| name           | type                         | description                                               |
+|----------------|------------------------------|-----------------------------------------------------------|
+| `mintAmount`   | `uint64`                     | The amount of funds being minted within this transaction. |
+| `mintAssetId`  | `byte[32]`                   | The asset ID of funds minted within this transaction.     |
+| `gasLimit`     | `uint64`                     | Gas limit for transaction                                 |
+| `scriptLength` | `uint16`                     | Script length, in instructions.                           |
+| `inputsCount`  | `uint8`                      | Number of inputs.                                         |
+| `outputsCount` | `uint8`                      | Number of outputs.                                        |
+| `txPointer`    | [TXPointer](./tx-pointer.md) | The location of the `Mint` transaction in the block.      |
+| `receiptsRoot` | `byte[32]`                   | Merkle root of receipts.                                  |
+| `script`       | `byte[]`                     | Script to execute.                                        |
```

Comment:
If we allow running any script, the `Mint` transaction may fail. Do we need to worry about that, or is it okay?=)

Reply:
Hmm good point - idk if there's really a way around this, especially if they use too much gas. Worst case they can put a change output to capture the fees if the script reverts, even if it's dust.

Reply:
removed the script and switched to intrinsic balance increase